### PR TITLE
DiD S12: Alternative campaign finish by letting the hero take the book

### PR DESCRIPTION
**Pull request by stevecotton**
On the third and subsequent playthroughs of Endless Night, Mal Keshar
has learned from the ending of SotA, and can leave the book to be
taken by the foolish hero's army.  This text doesn't handle the unit
that picks up the book being the hero themselves, but the hero seems
to be using the normal boss AI of staying near the castle so that
probably won't happen anyway.

He learns how to lichify level 3 necromancers at the start of the
first playthrough, which fits better with the backstory of the book.

This would close issues #3165 and #3167.

### CODE REVIEW

**Comment by CelticMinstrel** [comment]
Ooh an epilogue storyscreen, nice! I think this option also greatly improves the scenario; I never liked how the victory condition was to lose. It's still a bit of a strange condition, but it's one you could actively work towards. L

File: data/campaigns/Descent_Into_Darkness/scenarios/12_Endless_Night.cfg [hunk 1]
```diff
@@ -458,6 +565,197 @@
...
+                            [/message]
+
+                            [message]
+                                speaker=Mal Keshar
+                                message= _ "<small>(faking pain)</small> Aaargh!"
+                            [/message]
+
+                            [message]
+                                speaker=unit
+                                message= _ "Uuuuuh, I can't pull the skull off the book, but lich screams when I try."
```
Comment (stevecotton) [inline data/campaigns/Descent_Into_Darkness/scenarios/12_Endless_Night.cfg]: There's a word missing before "lich". I'm planning to push a version with this typo corrected and a few more translation hints added, that will be either today or tomorrow.

**Comment by Konrad22** [comment]
It sure feels nice to see these changes. :D

**Comment by GregoryLundberg** [comment]
Can @stevecotton update the change logs to resolve the conflicts?

This is against 1.14 and would also need to be ported to master. So the question is: does this just fix bugs or is the change sufficient to deny the merge to 1.14 and only put these changes into master? I'm for merging to both, but don't want to make the call without input.

**Comment by stevecotton** [comment]
I'll rebase after 1.14.5 has been released, not before because this can't be merged during string freeze.

**Comment by GregoryLundberg** [comment]
@stevecotton

Make the fixes and I'll merge and cherry pick this over the the new master branch, as well.

**Pull request merged**